app/views/layouts/temp_home.html.erb

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```

Comment:
HAML rather than ERB? I'm kind of with this guy: http://opensoul.org/2011/11/30/haml-the-unforgivable-sin/ :-\